repository/src/main/java/org/apache/atlas/glossary/GlossaryTermUtils.java

```diff
@@ -975,4 +975,20 @@ private void copyRelations(AtlasGlossaryTerm toGlossaryTerm, AtlasGlossaryTerm f
             }
         }
     }
+
+    protected String createQualifiedName(AtlasGlossaryTerm term) throws AtlasBaseException{
+        String qName = "";
+        if (!StringUtils.isEmpty(term.getQualifiedName())) {
+            //extract existing nanoid for category
+            String[] t1 = term.getQualifiedName().split("\\.");
```

Comment:
Can term's qualifiedName have "." character?

Reply:
Agree, changed for beta
https://github.com/atlanhq/atlas-metastore/pull/29